lib/rules/valid-model-definition.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileoverview Requires valid keys in model option.
+ * @author Alex Sokolov
+ */
+'use strict'
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const VALID_MODEL_KEYS = ['prop', 'event']
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'require valid keys in model option',
+      categories: ['essential'],
```

Comment:
`vue/valid-model-definition` does definitely make more sense for Vue 3. So either it should be in `vue3-essential` only, or in `vue3-essential` and `essential`.

Reply:
As far as I know, the model option has been removed in vue3. It is fixed with `modelValue` and `update:modelValue`. Am I misunderstanding something?

Reply:
Ah, again you're right.